.github/metadata.py

```diff
@@ -775,22 +787,18 @@ def format_github_value(value: Any, render_json=False) -> str:
 
     @cached_property
     def output_env_file(self) -> Path | None:
-        """Returns the `Path` to the environment file pointed to by the `$GITHUB_OUTPUT`
-        environment variable.
-        """
+        """Returns environment file `Path` pointed to by `$GITHUB_OUTPUT`."""
         output_path = None
         env_file = os.getenv("GITHUB_OUTPUT")
         if env_file:
             output_path = Path(env_file)
             assert output_path.is_file()
         return output_path
 
-    def save_metadata(self):
-        """Write data to the environment file pointed to by the `$GITHUB_OUTPUT`
-        environment variable.
-        """
+    def save_metadata(self) -> None:
+        """Write data to the environment file pointed to by `$GITHUB_OUTPUT`."""
         # Plain metadata.
-        metadata = {
+        metadata: dict[str, Any] = {
```

Comment:
I guess you can even be more precise with something like: `dict[str, tuple[Any, bool]]`

Reply:
The linter reported a suggested solution, but i tried it, and variations without getting it to quiet down. I settled for `Any` to push the PR for discussion. We can also quiet the line with a `# type: ignore` if nothing pleases `mypy`.

Reply:
I haven't tested the result of my suggestion so we will keep yours.